library/alloc/src/fmt.rs

```diff
@@ -163,7 +167,7 @@
 //! * `-` - Currently not used
 //! * `#` - This flag indicates that the "alternate" form of printing should
 //!         be used. The alternate forms are:
-//!     * `#?` - pretty-print the [`Debug`] formatting
+//!     * `#?` - pretty-print the [`Debug`] formatting (newline and indent)
```

Comment:
Maybe expand this to "adds linebreaks and indentation"?

Reply:
```suggestion
//!     * `#?` - pretty-print the [`Debug`] formatting (adds linebreaks and indentation)
```